synapse/lib/cell.py

```diff
@@ -728,8 +757,8 @@ async def saveHiveTree(self, path=()):
         return await self.cell.saveHiveTree(path=path)
 
     @adminapi()
-    async def getNexusChanges(self, offs, tellready=False):
-        async for item in self.cell.getNexusChanges(offs, tellready=tellready):
+    async def getNexusChanges(self, offs, tellready=False, wait=True):
```

Comment:
NB: This is to the point where we need docstrings for the behavior for future selves.